proto/kuksa/val/v2/val.proto

```diff
@@ -21,33 +21,45 @@ import "kuksa/val/v2/types.proto";
 
 service VAL {
   // Get the latest value of a signal
+  // If the signal exist but does not have a valid value
+  // a DataPoint where value is None shall be returned.
   //
   // Returns (GRPC error code):
   //   NOT_FOUND if the requested signal doesn't exist
   //   PERMISSION_DENIED if access is denied
+  //
+  //
   rpc GetValue(GetValueRequest) returns (GetValueResponse);
 
   // Get the latest values of a set of signals.
   // The returned list of data points has the same order as the list of the request.
+  // If a requested signal has no value a DataPoint where value is None will be returned.
   //
   // Returns (GRPC error code):
   //   NOT_FOUND if any of the requested signals doesn't exist.
   //   PERMISSION_DENIED if access is denied for any of the requested signals.
+  //
   rpc GetValues(GetValuesRequest) returns (GetValuesResponse);
```

Comment:
General question: Do we need to split this as well by `GetValuesId' and `GetValues`?

Reply:
The message `SignalID` supports both, so you can use whatever you like with the same RPC call.